kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/challenges/secondChallenge/ProductOrderService.java

```diff
@@ -12,17 +12,18 @@ public ProductOrderService(Order order) {
     }
 
     public String printCommunicateForBuyer() {
-        return String.format(INFO_FOR_BUYER, order.getSeller().getUserNick(), order.getQuantity(), order.getItem().getItemName(),
-                order.getSeller().getUserRealName(), order.getSeller().getUserAddress());
+        return String.format(INFO_FOR_BUYER, order.getSeller().getUserNick(), order.getQuantity(), order.getItem()
```

Comment:
popraw formatowanie :-) spacje, podział na linijki. Jeśli lista argumentów jest zbyd długa, to dobrze jest podawać argumenty w kolejnych linijkach w jednej kolumnie)